### ISSUE TITLE
cdms variables lost attribute (units. ect) after applying cdutil.YEAR()

### ISSUE THREAD

**Issue by chengzhuzhang** [CDAT/cdutil#43]
Attributes such as long_name, units.ect should stay. Might be a bug in the cdutil code...

**Comment by durack1** [comment]
@chengzhuzhang @jasonb5 I agree, when some calculations are performed (e.g. climatological averaging, with cdutil.year) all attributes, including units should be preserved. Obviously when another calculation, multiplication, addition etc occur this is not a reasonable assumption
